### PR TITLE
Add contributing guide and contributors file

### PR DESCRIPTION
**Pull request by klaudiagrz**
This pull request adds:
- Contributing guidelines
- Contributors file with the list of project contributors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    ```bash
+    git clone git@github.com:{YOUR_GITHUB_USERNAME}/{REPOSITORY_NAME}.git
+    ```
+
+3. Create a new branch:
+
+    ```bash
+    git checkout -b "{BRANCH_NAME}"
+    ```
```
Comment (klaudiagrz) [inline CONTRIBUTING.md:40]: I wonder if we need these two points if we provide a link to GitHub docs that explain how to work with forks.

Comment (0xAX) [reply CONTRIBUTING.md:40]: No, we can remove it as we have information

**Review by 0xAX — COMMENTED** [review]
LGTM. Could you please just add link to the README, to the contribution guide.

**Review by 0xAX — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/getting-started-with-git/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/using-github/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+**Working on your first pull request?** You can learn how from this free series [How to Contribute to an Open Source Project on GitHub](https://kcd.im/pull-request).
+
+## Create an issue 
+
+If you have any improvement ideas, notice a missing feature or a bug, create a GitHub issue by clicking **Issues -> New issue** in GitHub. Make sure to fill the issue template with the detailed description of the bug or suggested improvements. Provide proper argumentation and screenshots, if necessary.
```
Comment (klaudiagrz) [inline CONTRIBUTING.md:20]:
```suggestion
If you have any improvement ideas, notice a missing feature or a bug, create a GitHub issue by clicking **Issues -> New issue** in GitHub. Make sure to fill the issue template with a detailed description of the bug or suggested improvements. Provide proper argumentation and screenshots, if necessary.
```

**Pull request merged**